### ISSUE TITLE
[1.1.0] The DF instance is not compatible to terraform version 0.13

### ISSUE THREAD

**Issue by suchitgupta01** [terraform-google-modules/terraform-google-data-fusion#54]
### TL;DR

Module Version: 1.1.0

At root, I can see it accepts the terraform version to be >=0.13

`
terraform {
  required_version = ">= 0.13"
  required_providers {

    google = {
      source  = "hashicorp/google"
      version = ">= 3.53, < 5.0"
    }
  }

  provider_meta "google" {
    module_name = "blueprints/terraform/terraform-google-data-fusion/v1.1.0"
  }

}
`

https://github.com/terraform-google-modules/terraform-google-data-fusion/blob/master/versions.tf

But at the instance level, it is enforcing `~> 0.12.6`

https://github.com/terraform-google-modules/terraform-google-data-fusion/blob/master/modules/instance/versions.tf

### Expected behavior

Module 1.1.0 should work with terraform version >= 0.13

### Terraform Configuration

```hcl
module "data_fusion" {
  source  = "terraform-google-modules/data-fusion/google"
  version = "~> 1.1.0"

  name    = var.df_instance_name
}
```

### Terraform Version

```sh
0.13.7
```
